### DISCUSSION TITLE
Feature Request: Allow me to sort the display order of Global sets

### DISCUSSION THREAD

**Discussion by daltonrooney** [craftcms/cms#8529] [Ideas]
Globals are currently displayed in alphabetical order, which can sometimes be counterintutive. For example, if I have a global set called "Header" and a global set called "Footer", it makes more sense for them to be displayed in that order. A drag and drop handle in the Settings/Globals CP would help.

**Comment by KatieMFritz** [comment]
Potential workarounds in the meantime: number the global sets, use punctuation to pin some to the top

**Comment by bertoost** [comment]
👍 Yeah! I like to have this as well.. shouldn't that hard right? 😉 
So, why closed @brandonkelly ?

**Comment by brandonkelly** [comment]
@bertoost It’s open
